infrastructure/crashtests/example.html

```diff
@@ -1 +1,2 @@
+<!-- DO NOT SUBMIT -->
```

Comment:
If you change wptrunner I think we run all the infra tests anyway.

Reply:
Ahh I forgot. Thanks!